portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,13 +33,27 @@
 @WebServlet("/data")
```

Comment:
Perhaps for your class-level comment, you could update this one? Make sure it's clear to someone at a quick glance what this Servlet does!

Reply:
Done!